deployment/packer/template.js

```diff
@@ -4,7 +4,9 @@
         "branch": "",
         "aws_region": "",
         "aws_ubuntu_ami": "",
-        "stack_type": ""
+        "stack_type": "",
+        "postgresql_password": "",
```

Comment:
Looking at that code, it will attempt to access the `RDSPassword` key and throw an exception if it doesn't exist.